src/pages/styles.module.css

```diff
@@ -35,3 +37,38 @@
   height: 200px;
   width: 200px;
 }
+
+/**
+* For table of content items 
+*/
+.TOC {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-self: center;
+  margin: 1.5vh 0;
+  padding: 0;
+  
+  ul {
+    display: inline-block;
+    text-align: center;
+    list-style: none;
+    padding: 0;
+    margin: 0;
+  }
+  
+  /* ... The other nesting */
+}
+
+.TOC > ul > li {
+  margin-bottom: 1.5vh;
+  font-size: large;
+}
+
+.TOC > ul > li > ul {
+  text-align: center;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  font-size: smaller;
+}
```

Comment:
Make sure you add these elements to the .TOC class! So it will look something like:

```css
.TOC {
 /* styles */

  > ul {
    /* styles */
   
    > li {
      /* styles */
    
      > ul {
        /* styles */
      }
    }

  }
}